### ISSUE TITLE
Minor bug: character id's not being converted

### ISSUE THREAD

**Issue by kwazulueve** [eveseat/seat#314]
* **Problem**: character id's not translating to names at corporation/view/security/log/773499566
* **Expected**: Same like most of the other pages, conversion to names ;)
* **Logs / Screenshots / Proof**: https://i.imgur.com/Ryy5jEn.png
* **Version Info**: Debian 9, PHP 7.1, Apache, latest SeAT beta

**Comment by MrBMT** [comment]
Yeah this was being caused by a Javascript error as datatables was being included in the template but not used. I've submitted a PR to fix this, which I'm sure will be merged soon™

**Comment by kwazulueve** [comment]
Nice thanks
